src/modules/header/components/Header.js

```diff
@@ -195,7 +195,7 @@ export class Header extends MvuElement {
 					<div class="header__background">
 					</div>
 					<div class='header__search-container'>
-						<input id='input' @focus="${onInputFocus}" @blur="${onInputBlur}" @input="${onInput}" class='header__search' type="search" placeholder="" />          
+						<input id='input' placeholder='${translate('header_search_placeholder')}' @focus="${onInputFocus}" @blur="${onInputBlur}" @input="${onInput}" class='header__search' type="search" placeholder="" />          
```

Comment:
no test for this key (header_search_placeholder)

Reply:
there is one in header.provider.test?

Reply:
Meant in `Header.test.js` -> input should contain  placeholder attribute containing the correct i18n value